### PR TITLE
Fixing projection 2D thickness

### PR DESCRIPTION
**Pull request by cophus**
My math was all wrong before for the thickness tiling - it should be correct now!

### CODE REVIEW

File: py4DSTEM/process/diffraction/crystal_ACOM.py [hunk 1]
```diff
@@ -2098,6 +2123,8 @@ def calculate_strain(
 
     # Loop over all probe positions
     for rx, ry in tqdmnd(
+        # range(220,221),
+        # range(40,41),
```
Comment (sezelt) [inline py4DSTEM/process/diffraction/crystal_ACOM.py:2127]:
```suggestion
```

File: py4DSTEM/process/utils/single_atom_scatter.py [hunk 2]
```diff
@@ -46,6 +47,34 @@ def electron_scattering_factor(self, Z, gsq, units="A"):
         elif units == "A":
             return fe
 
+    def projected_potential(self, Z, R):
```
Comment (sezelt) [inline py4DSTEM/process/utils/single_atom_scatter.py:50]: What are the units of the returned potential?

**Review by sezelt — CHANGES_REQUESTED** [review]
looks good to me, I just suggested deleting some test lines and clarifying what units the potential are in

File: py4DSTEM/process/diffraction/crystal_ACOM.py (hunk 1 above)
Comment (cophus) [reply py4DSTEM/process/diffraction/crystal_ACOM.py:2127]: Good catch for both.  I tried to figure out the Lobato potential prefactors to properly set the output units, but no luck so far.

**Review by sezelt — APPROVED** [review]

**Pull request merged**